Formula/emacs-plus@28.rb

```diff
@@ -36,6 +36,7 @@ class EmacsPlusAT28 < EmacsBase
   depends_on "autoconf" => :build
   depends_on "gnu-sed" => :build
   depends_on "gnu-tar" => :build
+  depends_on "grep" => :build
```

Comment:
@d12frosted I wonder if it may be worth explicitly adding _all_ the autoconf GNU dependencies (perhaps in a future PR?) to cover this scenario better.

Reply:
I think yes. Ideally I would love to find a way to use homebrew `superenv` (e.g. default environment) which has everything properly configured. But path injection requires custom environment and hence all these issues 😢